### ISSUE TITLE
 any concrete dataset for the code?

### ISSUE THREAD

**Issue by vickersmith** [maeotaku/pytorch_multilabel_deep_learning_training#1]
hi,is there any concrete dataset for the code or any structual graphs for explanation? i counld't fully understand your code.for example ,a verfication code image that has 4 numbers(each varies from 0 to 9),4920 e.g, how can i convert this kind of images to your dataset type?Thanks very much. @maeotaku

**Comment by maeotaku** [comment]
Yes there are. You can find a plant dataset here from one of our papers: http://otmedia.lirmm.fr/LifeCLEF/GoingDeeperHerbarium/
The Herbarium255 dataset is the one you can use, which has a XML per image. The dataset is built based on such XMLs for each level of the hierarchy.
Paper: https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-017-1014-z

**Comment by maeotaku** [comment]
BTW right now the link is bronken but should be fixed since it is part of INRIA and they should not keep down their websites for long

**Comment by vickersmith** [comment]
if i train my own data image,i need to generate a xml  per image?but,how can the images be generated xml files?it seems that relevant code or information wasn't mentioned in your github.Thanks a lot @maeotaku

**Comment by maeotaku** [comment]
Hi, yes of course you will need to generate 1 xml per image with the metadata you need. To generate them you have to create your own code to create 1 xml tag per hierarchy level for each image. This is very common coding, but it will depend on how your data is structured.
You are correct that this information is not mentioned, I have to make some room to make a proper readme
